examples/huggingface-transformers/README.md

```diff
@@ -0,0 +1,113 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Hugging Face transformers - DeepSparse Inference Examples
+This directory contains examples of benchmarking, and serving Hugging Face transformers from the [hugging-face-transformers](https://github.com/huggingface/transformers)
+repository using the DeepSparse Engine. These examples can load pre-trained,
+sparsified models from [SparseZoo](https://github.com/neuralmagic/sparsezoo) 
+or you can specify your own transformer ONNX file
+
+## Installation
+The dependencies for this example can be installed using `pip`:
+```bash
+pip3 install -r requirements.txt
+```
+## Deepsparse pipeline Example
+
+The deepsparse pipeline integration provides a simple API dedicated to several 
+tasks, 
+following is an example using a pruned BERT model from sparsezoo for 
+Question-Answering task. 
+
+```python
+from deepsparse.transformers import pipeline
+
+onnx_filepath='zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-moderate'
```

Comment:
should we distinguish that this is a sparsezoo model stub?

Reply:
Will something like this work?

```
# SparseZoo model stub or path to ONNX file
onnx_filepath='zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-moderate'
```